### PR TITLE
Add redirect links to the contributor wiki

### PR DESCRIPTION
**Pull request by subramen**

### CODE REVIEW

File: docs/source/community/contribution_guide.rst
Comment (svekars) [inline docs/source/community/contribution_guide.rst:3]:
```suggestion
   For more guidelines, see the `Contribution Guide Wiki <https://github.com/pytorch/pytorch/wiki/The-Ultimate-Guide-to-PyTorch-Contributions>`_ 
```

File: docs/source/community/governance.rst
Comment (svekars) [inline docs/source/community/governance.rst:316]:
```suggestion
Wiki <https://github.com/pytorch/pytorch/wiki/The-Ultimate-Guide-to-PyTorch-Contributions>` for contribution
```

File: docs/source/community/contribution_guide.rst
Comment (subramen) [reply docs/source/community/contribution_guide.rst:3]: @albanD - what are your thoughts on an explicit deprecation callout v/s only linking out to current version?

File: CONTRIBUTING.md
Comment (albanD) [inline CONTRIBUTING.md:13]: Why indent everything by one here?

File: docs/source/community/contribution_guide.rst
Comment (albanD) [reply docs/source/community/contribution_guide.rst:3]: I don't think this we should share this link to get "more" information. It is intended as a replacement.
Deprecated or being moved to or outdated sounds like good phrasing.
WDYT @svekars ?

**Review by albanD — COMMENTED** [review]
Sounds good. Only small nits.

File: docs/source/community/contribution_guide.rst
Comment (svekars) [reply docs/source/community/contribution_guide.rst:3]: Is this page really deprecated? Did we do a thorough review of what's on that page and decided we will replace with what's in the wiki? If yes, sure.

Comment (subramen) [reply docs/source/community/contribution_guide.rst:3]: Yes, we did a thorough review - all of the info on this page has been updated on the wiki, and this page is not the current source of truth. Having the explicit callout will disambiguate this and prevent confusion for the user

**Review by albanD — APPROVED** [review]
The note is not rendering properly. Make sure this is fixed in the preview before merging.

File: docs/source/community/contribution_guide.rst
Comment (svekars) [reply docs/source/community/contribution_guide.rst:3]: Sounds good to me! Approving to unblock. @subramen do you have a plan to make further changes to the docs/source/community/contribution_guide.rst page so that we remove all irrelevant information?

**Review by svekars — APPROVED** [review]

File: docs/source/community/contribution_guide.rst
Comment (subramen) [reply docs/source/community/contribution_guide.rst:3]: @svekars  No further changes expected from me. I'm not aware of the archival policy for documentation, but perhaps we could follow that.

**Pull request merged**
